### PR TITLE
Github actions for cloud deployment

### PR DESCRIPTION
**Pull request by hanatgit**
This adds a Github action that will cause any PR created against `develop` branch to install, build, and run a trial deploy (`cdk diff`) of the webonary cloud backend to the `webonary.work` cloud infrastructure.  If the PR is successfully reviewed and merged into `develop`, then deployment to `webonary.work` will be started.

Similarly, a PR requested and merged into `master` will cause a trial deployment and full deployment to `webonary.org` cloud backend to take place.

### CODE REVIEW

File: .github/workflows/webonary-cloud-api-pr.yml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+env:
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set env to stage if PR is for merging to develop
```
Comment (johntseng) [inline .github/workflows/webonary-cloud-api-pr.yml]: This says set env to stage. Did you mean "dev"? line 29 says `DEPLOY_ENV=dev`

File: .github/workflows/webonary-cloud-api-pr.yml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ secrets.AWS_REGION }}
+
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: "12.X"
+          registry-url: "https://registry.npmjs.org"
+
+      - name: NPM 6.14.8
```
Comment (johntseng) [inline .github/workflows/webonary-cloud-api-pr.yml]: The latest npm 6.14 is 6.14.17. Consider using that instead.

**Review by johntseng — CHANGES_REQUESTED** [review]

File: .github/workflows/webonary-cloud-api-pr.yml (hunk 2 above)
Comment (hanatgit) [reply .github/workflows/webonary-cloud-api-pr.yml]: Nice catch. I think latest npm version 6 should be fine:

```
npm install -g npm@~6
```

File: .github/workflows/webonary-cloud-api-pr.yml (hunk 1 above)
Comment (hanatgit) [reply .github/workflows/webonary-cloud-api-pr.yml]: nice catch. corrected

**Review by johntseng — APPROVED** [review]

**Pull request merged**